### PR TITLE
Fix skip-host-cache deprecation warning

### LINKED ISSUE #921: Please use SET GLOBAL host_cache_size=0 instead
This warning appears when I restart MySQL docker instance:

> 2022-10-24T10:19:24.316236Z 0 [Warning] [MY-011068] [Server] The syntax '--skip-host-cache' is deprecated and will be removed in a future release. Please use SET GLOBAL host_cache_size=0 instead.

I don't maintain the `my.cnf` file, so I assume it is MySQL itself that is putting this `--skip-host-cache` variable in by default.

Thank you.

### LINKED ISSUE #990: Syntax '--skip-host-cache' is deprecated and will be removed in a future release. Please use SET GLOBAL host_cache_size=0 instead.
I cannot get rid of this warning and I tried to set global host cache size to 0 in a sql script.

### PR DESCRIPTION
**Pull request by rawr51919**
Fixes #990 
Fixes #921 
Also fixes https://github.com/hfagerlund/quiz_app/issues/9

### CODE REVIEW

File: 8.0/config/conf.d/docker.cnf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [mysqld]
-skip-host-cache
+SET GLOBAL host_cache_size=0
```
Comment (LaurentGoderre) [inline 8.0/config/conf.d/docker.cnf:2]: It doesn't work with the `SET GLOBAL`

This works for me:

```
[mysqld]
host_cache_size=0
skip-name-resolve

```

**Review by LaurentGoderre — CHANGES_REQUESTED** [review]

File: 8.0/config/conf.d/docker.cnf (hunk 1 above)
Comment (ghnp5) [reply 8.0/config/conf.d/docker.cnf:2]: Yeah, I found it strange that `SET GLOBAL` was here!! Never seen this syntax before, here.

Also, I'm curious if `host_cache_size` is needed, if `skip_name_resolve=1` ?

Comment (rawr51919) [reply 8.0/config/conf.d/docker.cnf:2]: @LaurentGoderre thanks for noticing that, will fix ASAP

Comment (rawr51919) [reply 8.0/config/conf.d/docker.cnf]: fixed, marking as resolved

**Comment by rawr51919** [comment]
The 8.0.34-debian PR CI job seems to keep failing on this PR. Can anyone look into why?

**Comment by LaurentGoderre** [comment]
It's failing because the apr source only has 8.0.35, not 8.0.34

```
apt-cache madison mysql-community-client                 
mysql-community-client | 8.0.35-1debian11 | http://repo.mysql.com/apt/debian bullseye/mysql-8.0 amd64 Packages
```

**Comment by LaurentGoderre** [comment]
Could you try rebasing on top of master?

**Comment by rawr51919** [comment]
> Could you try rebasing on top of master?

this is already on master, strange

**Comment by rawr51919** [comment]
there, updated to master

**Comment by rawr51919** [comment]
@LaurentGoderre what do you think? build's fixed, ready when you (and other maintainers) are

**Review by LaurentGoderre — APPROVED** [review]

**Review by yosifkit — APPROVED** [review]

**Comment by hfagerlund** [comment]
Thanks :+1:

**Pull request merged**